students/eric_grandeo/lesson03/mailroom.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+donors = [("Bill Gates", [653772.32, 12.17]),
+          ("Jeff Bezos", [877.33]),
+          ("Paul Allen", [663.23, 43.87, 1.32]),
+          ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+          ("Tim Cook", [1563.32, 8976.54])]
+
+prompt = "\n".join(("Welcome to the mailroom!",
+          "Please choose from below options:",
+          "1 - Send a thank you",
+          "2 - Create a report",
+          "3 - Quit",
+          ">>> "))
+
+
+def donor_list():
+    """Returns a list of the donor names from donors"""
+    donor_names = [i[0] for i in donors]
+    return donor_names
+    
+
+def add_donation(name, donation):
+    """
+    Appends the donation to the donation list in the donor tuple
+    
+    Parameters:
+    name(str): name of the donor
+    donation(int): value of the donation
+    
+    Returns:
+    list with new donation added
+    """
+    for i in donors:
+        if name in i:
+            place = donors.index(i)
+            thankyou_email(name, donation)
+            return donors[place][1].append(donation)
```

Comment:
your return implies that you're returning a value - append() always returns a None

I believe you tried to combine two statements here - but this produces not-clean code.

you can either break into two 
```
append(donation)
return
```
or just use a break statement after append()


I would also recommend to just have this function do both, add and update donors.

right now you're looping over the full list to see if donors exists, if doesn't exist then you add a donor with no donation. then you're looping again, to update the donor.